src/PhpWord/Shared/ZipArchive.php

```diff
@@ -162,13 +163,16 @@ public function open($filename, $flags = null)
      * Close the active archive.
      *
      * @return bool
-     *
-     * @codeCoverageIgnore Can't find any test case. Uncomment when found.
      */
     public function close()
     {
         if (!$this->usePclzip) {
-            if ($this->zip->close() === false) {
+            try {
+                $result = @$this->zip->close();
+            } catch (Throwable $e) {
+                $result = false;
+            }
+            if ($result === false) {
```

Comment:
```suggestion
```

Reply:
You can remove these lines.

Reply:
No, your suggestion changes the program logic. We should get `false` for the result of `close`. Php8 changed that, and I have filed a bug with Php about it. For Php7, we still get false, and we want to throw an exception (to avoid a compatibility break for Php7, which is what this change is all about). We also want to throw an exception when Php8 (erroneously) throws an exception for `close`. The way to handle both is to evaluate `$result` outside the try-catch, as I have currently coded it.

Reply:
> No, your suggestion changes the program logic. We should get `false` for the result of `close`. Php8 changed that, and I have filed a bug with Php about it. For Php7, we still get false, and we want to throw an exception (to avoid a compatibility break for Php7, which is what this change is all about). We also want to throw an exception when Php8 (erroneously) throws an exception for `close`. The way to handle both is to evaluate `$result` outside the try-catch, as I have currently coded it.

I would like to ask, how can I get Progi1984 to review my code?

Reply:
There is a big backlog, and I think he is being very careful and methodical in how he's trying to catch up. I'm sure he will get to your PR, perhaps not as quickly as you would like, but hopefully not too long.